cpp/include/cudf/detail/utilities/device_operators.cuh

```diff
@@ -128,14 +129,15 @@ struct DeviceMin {
                               !cudf::is_fixed_point<T>()>* = nullptr>
   static constexpr T identity()
   {
-    return std::numeric_limits<T>::max();
+    if constexpr (cudf::is_chrono<T>()) return std::numeric_limits<T>::max();
```

Comment:
I suspect my earlier comment got buried. This still makes me very apprehensive I'd still like to see it understood or fixed before merging.

Reply:
Ok, I will try to track down the cause.

Reply:
Fixed.